azext_iot/tests/iothub/test_iot_messaging_int.py

```diff
@@ -217,7 +259,11 @@ def test_uamqp_device_messaging(self):
             expect_failure=True,
         )
 
-    def test_device_messaging(self):
+    @pytest.mark.skipif(
+        not validate_min_python_version(3, 4, exit_on_fail=False),
```

Comment:
The minimum python we support now is 3.6 so we don't need this skipif mark (or others targeting 3.5 and below).